tests/unit/amo/components/TestConfirmButton.js

```diff
@@ -1,17 +1,12 @@
 import * as React from 'react';
+import userEvent from '@testing-library/user-event';
 
-import Button from 'amo/components/Button';
-import ConfirmButton, {
-  ConfirmButtonBase,
-  extractId,
-} from 'amo/components/ConfirmButton';
-import ConfirmationDialog from 'amo/components/ConfirmationDialog';
+import ConfirmButton, { extractId } from 'amo/components/ConfirmButton';
 import {
-  applyUIStateChanges,
-  createFakeEvent,
   dispatchClientMetadata,
   fakeI18n,
-  shallowUntilTarget,
+  render as defaultRender,
+  screen,
 } from 'tests/unit/helpers';
 
 describe(__filename, () => {
```

Comment:
I think we can get rid of `getProps` below. Much of it isn't valid anyway. We no longer want/need to pass in `i18n` or `store`, and `onConfirm` should default to `jest.fn()` not `sinon.stub()`. I would just add the ones we need as arguments to `render` with default values.